learning_journal/templates/new-entry.html

```diff
@@ -1,3 +1,9 @@
-<h1>new-entry.html</h1>
-<h1>More crappy html</h1>
-<h2><a href='/'>Back to Home</a></h2>
+<!DOCTYPE html>
+{% extends "bootstatmenu.jinja2" %}
+{% block body %}
+
+  <h1>new-entry.html</h1>
+  <h1>More crappy html</h1>
+  <h2><a href='/'>Back to Home</a></h2>
+
+{% endblock %}
```

Comment:
now that you have the `.jinja2` version of this file, you don't need to keep the `.html` version.